content/psy_recommend/psy_recommend_2.md

```diff
@@ -0,0 +1,43 @@
+title: Рекомендации по приобретению уверенности
+slug: psy2
+category: psy_recommend
+tags: [_nav,_category_header]
+---
+
+Рекомендации по приобретению уверенности
+
+1. Уважайте себя и других.
+2. Отвечайте на негативную критику дружескими вопросами типа: «Что Вы имеете в виду?», «Что Вы хотите этим сказать?» и т.д.
+3. Добивайтесь того, чтобы окружающие хотели сотрудничать с Вами.
+4. Не стремитесь быть всегда услужливыми, имейте смелость сказать «нет».
+5. Будьте последовательны в своих решениях и действиях.
+6. Слушайте других и себя.
+7. Обращайте внимание на тон Вашего голоса.
+8. Обращайте внимание на язык Вашего тела.
+9. Разговаривая с людьми, общайтесь с ними, а не читайте лекции.
+10.  Не играйте в силовые игры, не прибегайте к нажиму.
```

Comment:
Leave exactly _one_ space between the dot and a text in all the subsequent list items.